package.json

```diff
@@ -33,11 +33,10 @@
     "@ckeditor/ckeditor5-dev-tests": "^53.1.0",
     "@ckeditor/ckeditor5-dev-web-crawler": "^53.1.0",
     "@ckeditor/ckeditor5-inspector": "^5.0.0",
-    "@ckeditor/ckeditor5-mermaid": "ckeditor/ckeditor5-mermaid#v0.0.5",
+    "@ckeditor/ckeditor5-mermaid": "ckeditor/ckeditor5-mermaid#v0.0.6",
     "@ckeditor/ckeditor5-react": "^9.5.0",
     "@inquirer/prompts": "^7.5.0",
     "@listr2/prompt-adapter-inquirer": "^2.0.16",
-    "@ocular-d/vale-bin": "^2.29.1",
```

Comment:
So also `@ocular-d/vale-bin` should be removed from `pnpm-workspace.yaml`.

Also, we should remove `docs:lint` and `docs:vale` scripts since the binary will not be available anymore.